### DISCUSSION TITLE
Vote to promote @ryanmoran to maintainer

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#96] [Content Team]
I have contributed the blog implementation, a couple of blog posts, some docs, CSS/html tweaks, and maintain the current ROADMAP.md in the community repo. With those contributions, I'd love to join the maintainers team and help to guide the efforts for content in the Paketo org.

@paketo-buildpacks/content-maintainers can I get a 👍 / 👎 ?

**Comment by ForestEckhardt** [comment]
@ryanmoran I believe that you have a super majority.
